tests/components/brother/conftest.py

```diff
@@ -96,7 +96,7 @@ def mock_unload_entry() -> Generator[AsyncMock, None, None]:
 
 
 @pytest.fixture
-def mock_brother_client() -> Generator[AsyncMock, None, None]:
+def mock_brother_client() -> Generator[AsyncMock]:
```

Comment:
```suggestion
def mock_brother_client() -> Generator[Mock]:
```

For this one, it would be really good to use `Generator[MockOf[Brother]]` but it's not available yet.